src/backend/marsha/core/views.py

```diff
@@ -588,6 +543,101 @@ def post(self, request, *args, **kwargs):
         return self.render_to_response(self.get_context_data())
 
 
+@method_decorator(csrf_exempt, name="dispatch")
+@method_decorator(xframe_options_exempt, name="dispatch")
+class LTIRespondView(TemplateResponseMixin, View):
+    """LTI view called to respond to a consumer.
+
+    ie. after a deep linking content selection.
+
+    """
+
+    template_name = "core/form_autosubmit.html"
+
+    # pylint: disable=unused-argument
+    def post(self, request, *args, **kwargs):
+        """Respond to POST request.
+
+        Renders a form autosubmitted to a LTI consumer with signed parameters.
+
+        Parameters
+        ----------
+        request : Request
+            passed by Django
+        args : list
+            positional extra arguments
+        kwargs : dictionary
+            keyword extra arguments
+        Returns
+        -------
+        HTML
+            generated from applying the data to the template
+
+        """
+        try:
+            jwt_token = AccessToken(self.request.POST.get("jwt", ""))
+            jwt_token.verify()
+
+            lti_select_form_data = jwt_token.payload.get("lti_select_form_data")
+            content_item_return_url = lti_select_form_data.get(
+                "content_item_return_url"
+            )
+
+            urlparse(content_item_return_url)
```

Comment:
You don't set the return of this function in a variable ? Is it needed ?

Reply:
I'ts only here to assert that content_item_return_url is really an url.
But it is overkill and a little cryptic, I'll remove it as it checked later in the client.sign function.